app/controllers/payment-links/post-information-controller.js

```diff
@@ -12,8 +13,12 @@ module.exports = (req, res) => {
 
   updatedPageData.paymentLinkTitle = req.body['payment-link-title']
   updatedPageData.paymentLinkDescription = req.body['payment-link-description']
+  updatedPageData.serviceNamePath = req.body['service-name-path']
```

Comment:
What have we decided for the service name. I reckon the same normalisation rules apply as for product name. May we should create a separate function that we can reuse as we type the title and then again when we validate the input fields after an update?